com.unity.render-pipelines.high-definition/Documentation~/Decal-Projector.md

```diff
@@ -14,29 +14,33 @@ The Decal Projector includes a Scene view representation of its bounds and proje
 
 * A box that describes the 3D size of the projector; the projector draws its decal on every Material inside the box.
 
-* An arrow that indicates the direction the projector faces.
+* An arrow that indicates the direction the projector faces. This arrow is on the pivot point.
```

Comment:
The base of this arrow is on the pivot point.

^^
Assumed its the base, ignore this comment if it isn't